app/models/user.rb

```diff
@@ -38,6 +38,7 @@ class User < ActiveRecord::Base
   has_many :permissions,       :through => :filters
   has_many :cached_usergroup_members
   has_many :widgets, :dependent => :destroy
+  has_many :ssh_keys, :dependent => :destroy
```

Comment:
@ares : Good point. `owner` is currently now allowed in host's jail. I do remember, that there were some concerns about this. Can you think of some?